src/api/client.ts

```diff
@@ -1,9 +1,9 @@
 import axios from 'axios';
-
-import { KWITTER_API_BASE_URL } from '../config';
+//import { KWITTER_API_BASE_URL } from '../config';
 
 const kwitterApiClient = axios.create({
-  baseURL: KWITTER_API_BASE_URL
+  //removed KWITTER_API_BASE_URL temporary as it throws an error
+  baseURL: `"http://localhost:3001"`
```

Comment:
Changes to this file need to be reverted to use the value pulled in from `.env` files